libsodium-sys/src/crypto_secretbox_xsalsa20poly1305.rs

```diff
@@ -16,12 +16,26 @@ extern {
         mlen: c_ulonglong,
         n: *const [u8; crypto_secretbox_xsalsa20poly1305_NONCEBYTES],
         k: *const [u8; crypto_secretbox_xsalsa20poly1305_KEYBYTES]) -> c_int;
+    pub fn crypto_secretbox_detached(
```

Comment:
Ah my mistake, that file didn't exist when this PR was opened, but it does exist now. I'll rebase and move my declarations into there. (Do you have a preference for history-is-immutable merge commits vs rebase-and-squash?)